pkg/agent/qrm-plugins/cpu/dynamicpolicy/cpuadvisor/helper.go

```diff
@@ -316,3 +317,15 @@ func (ci *CalculationInfo) GetCPUSet(entry, subEntry string, b BlockCPUSet) (mac
 	}
 	return cpusets, nil
 }
+
+func IsBlockOfRelcaim(blockEntryNames [][]string) bool {
```

Comment:
should we move those functions to one package? not insist, you can just resolve this comment if not necessary @csfldf @nightmeng